django_react_proj/users-fe/src/components/ConfirmRemovalModal.js

```diff
@@ -1,53 +1,56 @@
-import React, { Component, Fragment } from "react";
+import React, { useState, Fragment } from "react";
 import { Modal, ModalHeader, Button, ModalFooter } from "reactstrap";
 import axios from "axios";
 import { API_URL } from "../constants";
 
-class ConfirmRemovalModal extends Component {
-  state = {
-    modal: false
-  };
+const ConfirmRemovalModal = (props) => {
```

Comment:
Try to destructure props (instead of `(props)` - `({id, deleteUser})`) and then validate them with PropTypes https://pl.reactjs.org/docs/typechecking-with-proptypes.html#function-components